test/Conversion/TorchToLinalg/basic.mlir

```diff
@@ -95,6 +95,38 @@ func.func @torch.aten.Int.Tensor$non_zero_rank(%arg0: !torch.vtensor<[?,?],si64>
 
 // -----
 
+// CHECK-LABEL:     func.func @torch.aten.Int.Tensor$zero_rank$byte_dtype
+// CHECK-SAME:          (%[[ARG:.*]]: !torch.vtensor<[],ui8>) -> !torch.int {
+// CHECK:               %[[I:.*]] = torch_c.to_builtin_tensor %[[ARG]] : !torch.vtensor<[],ui8> -> tensor<i8>
+// CHECK:               %[[C1_I64:.*]] = arith.constant 1 : i64
+// CHECK:               %[[C0:.*]] = arith.constant 0 : index
+// CHECK:               %[[EXTRACT:.*]] = tensor.extract %[[I]][] : tensor<i8>
+// CHECK:               %[[RES:.*]] = arith.extsi %[[EXTRACT]] : i8 to i64
```

Comment:
This doesn't seem right to me. In the case of `ui8`, the op generated for the extension should be `arith.extui`